src/features/content-explorer/item-list/ItemList.js

```diff
@@ -188,8 +190,30 @@ const ItemList = ({
         };
     }
 
+    const addAutoSizer = WrappedComponent => {
```

Comment:
Thoughts on possibly calling this `withAutoSizer`?  I'm unsure if we tend to use `add` for components although I'm not great on knowing the specifics of when to use different prefixes

Reply:
I went ahead and updated it to `withAutoSizer()`. Initially, I hesitated because it's an internal method and not a usable function by other components. However, it is acting like an HOC so from a code readability standpoint makes sense to use `withAutoSizer()`.